ddtrace/tracer.py

```diff
@@ -43,8 +44,8 @@ def __init__(self):
         # A hook for local debugging. shouldn't be needed or used in production
         self.debug_logging = False
 
-        # globally set tags
-        self.tags = {}
+        # globally set tags (`lang` and `lang.version` are set by default)
+        self.tags = {"lang": "python", "lang.version": platform.python_version()}
```

Comment:
instead of making this call here, I'd say to move everything in the `compat.py` library so that:
```python
# compat.py
PYTHON_VERSION = platform.python_version()
PYTHON_INTERPRETER = platform.python_implementation()
```
so they're constant for this interpreter execution. Then in your tracer you can:
```python
self.tags = {
  "lang": "python", 
  "lang.version": "{}-{}".format(PYTHON_VERSION, PYTHON_INTERPRETER)
}
```
This should result in a string like `2.7.12-CPython` so we have also the used interpreter on that meta (it may be `PyPy` or others in the future). Or maybe it could be a `lang.interpreter` field if we want to keep them separated.

Don't know if that makes sense now, but I think it could be interesting for the future.